<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Galaxy Game - Space Cowboys</string>
  </property>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>85</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Lopeta</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSaved">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelLost">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGalaxy">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>62</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
